pages/documentation/simulation/powerflow/openlf.md

```diff
@@ -10,6 +10,75 @@ PowSyBl OpenLoadFlow is an open-source power flow implementation in Java provide
+
+As the \\(\Pi\\) model is created from IIDM grid modelling that locates its ratio and phase tap changers in side 1, \\(A_2\\) and \\(R_2\\) are always equal to zero and \\(1\\). In case of a branch with voltage or phase control, the \\(\Pi\\) model becomes an array. See below our model:
+
+![Pi model](img/pi-model.svg){: width="50%" .center-image}
+
+### AC flows computing
+
+TO DO
+
+### DC flows computing
+
+The DC flows computing relies on several classical assumptions to build a model where the active power flowing through a line depends linearly from the voltage angles at its ends.
+In this simple model, reactive power flows and active power losses are totally neglected. The following assumptions are made to ease and speed the computations:
+- The voltage magnitude is equal to 1 per unit at each bus,
+- The conductance \\(G_{i,j}\\) of each line \\((i,j)\\) is neglected, only the susceptance \\(B_{i,j}\\) is considered,
+- The voltage angle difference between two adjacent buses is considered as very small.
+
+Therefore, the power flows from bus \\(i\\) to bus \\(j\\) following the linear expression:
+
+$$ P_{i,j} = \frac{\theta_i-\theta_j+A_{i,j}}{X_{i,j}} $$
+
+Where \\(X_{i,j}\\) is the reactance of the line \\((i,j)\\), \\(\theta_i\\) the voltage angle at bus \\(i\\) and \\(A_{i,j}\\) is the phase angle shifting on side \\(j\\).
+
+DC flows computing give a linear grid constraints system.
+The variables of the system are, for each bus, the voltage angle \\(\theta\\).
+The constraints of the system are the active power balance at each bus, except for the slack bus.
+The voltage angle at slack bus is set to zero.
+Therefore the linear system is composed of \\(N\\) variables and \\(N\\) constraints, where \\(N\\) is the number of buses in the network.
+
+The linear matrix \\(J\\) of this system satisfied:
+
+- If \\(i\\) is the slack bus:
+
+	$$J_{i,i} = 1$$
+- Else, let \\(v(i)\\) be the buses linked to \\(i\\) in the network graph:
+
+	$$J_{i,i} = \sum_{j \in v(i)} \frac{1}{X_{i,j}}.$$
+
+	$$ \forall j \in v(i), \quad J_{i,j} = - \frac{1}{X_{i,j}}.$$
+- All other entries of \\(J\\) are zeroes.
+
+The right-hand-side \\(b\\) of the system satisfied:
+
+- If \\(i\\) is the slack bus:
+
+	$$b_{i} = 0$$
+- Else, let \\(v(i)\\) be the buses linked to \\(i\\) in the network graph:
+
+	$$b_{i} = P_i - \sum_{j \in v(i)} \frac{A_{i,j}}{X_{i,j}}.$$
+	
+	Where \\(P_i\\) is the injection at bus \\(i\\).
+
+This linear system is resumed by:
+$$ J\theta = b $$
+Note that the vector \\(b\\) of right-hand sides is linearly computed from the given injections and phase-shifting angles.
+
+To solve this system, the classic technic is to use the LU matrices decomposition $$ J = LU $$.
+Hence matrices \\(L\\) and \\(U\\) are obtained, it is quite easy to solve the grid constraints linear system.
+The solution of this system gives the voltage angles corresponding to injections at each bus (minus the slack bus) and phase shifting angles.
```

Comment:
In case of a load flow, the user needs to provide the phase shifting angle value for each PST. In case of a sensitivity computation according to injections, there is no need to provide phase shifting angle values.

Reply:
I am not sure to get what you mean in our sentence.

Reply:
Let me reformulate this sentence :
The grid equations system takes as variables the voltage angles. The right-hand side  of the system is computed knowing the injection at each bus and the phase shifting angle of each PST.

Hence by solving the system using LU decomposition, you can compute the voltage angles by giving as data the injections and the phase shifting angles.

In the special case of a sensitivity computation according to an increase of injection at a bus, the right-hand side is independant from PST angles.

Reply:
So phase shifting angles are not outputs, and it seems to with our sentence, no ? The reformulation is better!

Reply:
Yes they are not outputs.